src/domain/orders/details/returns/index.js

```diff
@@ -30,12 +32,27 @@ const ReturnMenu = ({ order, onReturn, onDismiss, toaster }) => {
 
   const [allItems, setAllItems] = useState([])
 
+  const [showCustomPrice, setShowCustomPrice] = useState({
+    standard: false,
+    return: false,
+  })
+  const [customOptionPrice, setCustomOptionPrice] = useState({
+    standard: 0,
```

Comment:
And this one?

Reply:
Also used, for storing a custom price for shipping if needed